cmd/notary/tuf.go

```diff
@@ -546,3 +587,30 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	}
 	return defaultServerURL
 }
+
+func getTrustPinning(config *viper.Viper) (trustpinning.TrustPinConfig, error) {
+	var ok bool
+	// Need to parse out Certs section from config
+	certMap := config.GetStringMap("trust_pinning.certs")
+	resultCertMap := make(map[string][]string)
+	for gun, certSlice := range certMap {
+		var castedCertSlice []interface{}
+		if castedCertSlice, ok = certSlice.([]interface{}); !ok {
+			return trustpinning.TrustPinConfig{}, fmt.Errorf("invalid format for trust_pinning.certs")
```

Comment:
We can't return nil with this function return signature since we return the struct itself instead of a pointer -- I can change this function to return `*trustpinning.TrustPinConfig` and a nil pointer for error cases if we prefer?